client/gui_rpc_server_ops.cpp

```diff
@@ -804,7 +804,11 @@ void handle_lookup_account_poll(GUI_RPC_CONN& grc) {
             grc.lookup_account_op.error_num
         );
     } else {
-        grc.mfout.printf("%s", grc.lookup_account_op.reply.c_str());
+        const char *p = grc.lookup_account_op.reply.c_str();
+        const char *q = strstr(p, "<account_out"); 
+        if (!q) q = strstr(p, "<error");
+        if (!q) q = "<account_out/>\n"; 
```

Comment:
The error reporting is inconsistent. `get_project_config_poll` returns errors like this:
```
<project_config>
    <error_num>%d</error_num>
</project_config>
```
No special handling required here. `lookup_account_poll` returns
```
<account_out>
  <error_num>%d</error_num>
</account_out>
```
in case of a client-side error (e.g. GetHostByName) and
```
<error>
  <error_num>%d</error_num>
  <error_msg>%s</error_msg>
</error>
```
in case of a server-side error (e.g. DBNotFound). `get_all_projects_list` again does not have any error reporting at all and returns an empty response in the unlikely case that all_projects_list.xml cannot be read.

This is why there's special handling of `<error>` in `lookup_account_poll`.